pandas/tseries/tests/test_tslib.py

```diff
@@ -608,6 +608,9 @@ def test_parsers(self):
         self.assertTrue(result1 is tslib.NaT)
         self.assertTrue(result1 is tslib.NaT)
 
+    def test_datetime_bool(self):
+        self.assertTrue(to_datetime(False) is tslib.NaT)
```

Comment:
also with ``to_datetime(True)``, not sure what this should do.

Further test with a list of bools & NaT, e.g. ``[ True, False, NaT ]``